FFMpegCore/FFMpeg/Arguments/SetMirrorVideo.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FFMpegCore.Arguments
+{
+    public class SetMirrorVideo : IArgument
```

Comment:
Should not be a regular IArgument but instead a VideoFilter argument (see `ScaleArgument` for example of this)

Reply:
I little remade. But left IArgument, since flip does not have a key and therefore IVideoFilter does not fit.

https://duxyng.wordpress.com/2013/04/07/rotateflip-video-with-ffmpeg/

Reply:
Any video filter (-vf) argument MUST be a `IVideoFilterArgument` and if you want it added, you should also provide a method for the fluent argument builder.
Again, see [ScaleArgument](https://github.com/rosenbjerg/FFMpegCore/blob/master/FFMpegCore/FFMpeg/Arguments/ScaleArgument.cs)

Reply:
It needs to be a `IVideoFilterArgument` since ffmpeg only accepts a single -vf.
Just set the Key to vflip/hflip and leave the value as an empty string and I'll make that work when merging